public/docs/ganache/using.md

```diff
@@ -0,0 +1,64 @@
+# Working with Ganache
+
+[Ganache](/ganache) is an application that generates a personal blockchain you can use to run tests, execute commands, deploy contracts, and inspect state.
+
+![Ganache logo](/docs/ganache/images/ganache-logo.png)
+
+It is available in both a graphical format and as a terminal application (which was previously known as the TestRPC). Ganache is available for Windows, macOS, and Linux.
+
+## Installation
+
+### Ganache
+
+Installation of Ganache is only a few steps. [Download](https://github.com/trufflesuite/ganache/releases) the appropriate version for your OS:
+
+* Windows: `Ganache-a.b.c.appx`
```

Comment:
Nit picky, but can we make these `Ganache-*.appx`, etc? This way we don't have to change the docs if we choose to change the release format.